akka-cluster/src/main/scala/akka/cluster/routing/ClusterRouterConfig.scala

```diff
@@ -28,14 +28,23 @@ import com.typesafe.config.ConfigFactory
 
 import scala.annotation.tailrec
 import scala.collection.immutable
+import scala.collection.JavaConverters._
 
 object ClusterRouterGroupSettings {
+  @deprecated("useRole has been replaced with useRoles")
```

Comment:
`@deprecated("...", since = "2.5.4")` (and same for all the deprecated annotations)